### DISCUSSION TITLE
PaddleOCR/latest/datasets/table_datasets

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#14185] [Q&A]
# PaddleOCR/latest/datasets/table_datasets

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/latest/datasets/table_datasets.html

**Comment by Garry1024** [comment]
good

**Comment by RunSongham** [comment]
very  good!! 

thank you!
